PZ/WebClient/tests.py

```diff
@@ -1,3 +1,103 @@
 from django.test import TestCase
+from .models import *
+from .Connector import *
 
 # Create your tests here.
+url_local = 'http://localhost:8002/'
+url_resources = 'http://localhost:8002/resources/'
+url_measurements = 'http://localhost:8002/measurements/'
+
+class ConnectorTests(TestCase):

```

Comment:
The tests should be independent of the machine. Do not use guid or other data from your local database.
Mocking is good practise:
http://stackoverflow.com/questions/15753390/python-mock-requests-and-the-response